docs/resources/supported_operations.md

```diff
@@ -0,0 +1,1072 @@
+       ShuffleChannels                    Supported       Supported**      Not Supported     Supported       
+       Sign                               Supported       Supported**      Not Supported     Supported       
+       Sin                                Supported       Supported**      Not Supported     Supported       
+       Sinh                               Supported       Supported**      Not Supported     Supported****
+       SimplerNMS                         Supported       Supported**      Not Supported     Not Supported   
+       Slice                              Supported       Supported***     Supported         Not Supported   
+       SoftMax                            Supported       Supported***     Not Supported     Supported       
+       Softplus                           Supported       Supported**      Not Supported     Supported       
+       Softsign                           Supported       Supported**      Supported         Not Supported   
+       SpaceToDepth                       Not Supported   Supported**      Not Supported     Supported*     
+       SpatialTransformer                 Not Supported   Supported**      Not Supported     Not Supported   
+       Split                              Supported       Supported***     Supported         Supported       
+       Squeeze                            Supported       Supported**      Supported         Supported       
+       StridedSlice                       Supported       Supported**      Not Supported     Supported*     
+       Tan                                Supported       Supported**      Not Supported     Supported****
+       TensorIterator                     Not Supported   Supported        Supported         Supported       
+       Tile                               Supported**     Supported***     Not Supported     Supported       
+       TopK                               Supported       Supported**      Not Supported     Supported****
+       Unpooling                          Supported       Not Supported    Not Supported     Not Supported   
+       Unsqueeze                          Supported       Supported**      Supported         Supported       
+       Upsampling                         Supported       Not Supported    Not Supported     Not Supported   
+      =================================  ==============  ===============  ================  ==================
+      
+      |   `*` - support is limited to the specific parameters. Refer to "Known Layer Limitations" section for the device :doc:`from the list of supported <openvino_docs_OV_UG_supported_plugins_Supported_Devices>`.
+      |   `**` - support is implemented via :doc:`Extensibility mechanism <openvino_docs_Extensibility_UG_Intro>`.
+      |   `***` - supports NCDHW layout.
+      |   `****` - support is implemented via runtime reference.
+
+
+   .. tab-item:: Operations by Frameworks
+
+      .. tab-set::
+      
+         .. tab-item:: Caffe
```

Comment:
Also, I would like to change the order of frameworks here and put deprecated frameworks in the last order (Caffe, Kaldi, MXNet). So the final order should be: TensorFlow, TensorFlow 2 Keras, TensorFlow Lite, ONNX, PaddlePaddle, Caffe, Kaldi, MXNet.

@mvafin, will you put PyTorch supported operations into this doc?

Reply:
Yes, seems like a good idea